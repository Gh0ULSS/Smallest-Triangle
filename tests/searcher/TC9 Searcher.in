20.20.82, -21.94
-23.67, -27.14
25.58.48, 49.58
7. , -27.26
12.35, -42.23
35.78, 37.45
29.76, -27.66
34.26, -12.25.47
19.51, -45.85, 15.23