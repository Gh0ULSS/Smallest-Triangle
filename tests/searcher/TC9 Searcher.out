read 4 points
12.35, -42.23
29.76, -27.66
-23.67, -27.14
This is a triangle